docs/manuals/gui/viselements/charts/bubble.md

```diff
@@ -1,3 +1,178 @@
-## Bubble charts
+# Bubble charts
 
-TODO
+Bubble charts are a type of [scatter chart](scatter.md) where marker sizes and colors are depending
+on the data: columns in the data set are used to retrieve the values for the color and size of each
+data point representation, using the [*marker*](../chart.md#p-marker) property of the chart control.
+
+The [*mode*](../chart.md#p-mode) property needs to be set to "markers" so that only
+the data points get rendered.
+
+## Key properties
+
+| Name            | Value           | Notes   |
+| --------------- | ------------------------- | ------------------ |
+| [*type*](../chart.md#p-type)      | `scatter`          | Default value  |
+| [*mode*](../chart.md#p-mode)      | `markers`          |  |
+| [*x*](../chart.md#p-x)            | x values           |  |
+| [*y*](../chart.md#p-y)            | y values           |  |
+| [*marker*](../chart.md#p-marker)  | dictionary  | `size` can be set to a column name in *data*.<br/>`color` can be set to a column name in *data* that holds color names or values.<br/>`symbol` can be set to a column name in *data* that holds symbol names (see [Symbol names](https://plotly.com/javascript/reference/scatter/#scatter-marker-symbol) for details).<br/>`opacity` can be set to a column name in *data* that holds an opacity value between 0 and 1.  |
+
+## Examples
+
+### Simple bubble chart {data-source="gui:doc/examples/charts/bubble-simple"}
+
+A simple use case for bubble charts is a data set where data points have a related color,
+size, and opacity to be applied for their representation.
+
+Here is a small example of this:
+
+```py
+data = {
+    "x": [1, 2, 3],
+    "y": [1, 2, 3],
+    "Colors": ["blue", "green", "red"],
+    "Sizes": [20, 40, 30],
+    "Opacity": [1, .4, 1]
```

Comment:
Opacities ? bizarre ...

Reply:
However correct. And it clarifies the distinction between the property name and the column name.